packages/common/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@divviup/common",
+  "version": "0.1.0",
+  "description": "",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "source": "src/index.ts",
+  "scripts": {
+    "clean": "rm -rf dist/*",
+    "build:clean": "npm run clean && npm run build",
+    "build": "parcel build",
+    "docs": "typedoc src",
+    "test": "ts-mocha \"src/**/*.spec.ts\" -p ./tsconfig.json",
+    "lint": "eslint src --ext .ts && prettier -c src",
+    "format": "prettier -w src",
+    "test:coverage": "nyc --cwd=../.. --no-clean --reporter=lcov --reporter=text --reporter=json npm test",
+    "ci": "npm run lint && npm run test:coverage",
+    "benchmark": "ts-node scripts/benchmarks.ts"
```

Comment:
nit: this command is defined on several packages, but we only have the one benchmark script hanging off of the top-level package.